include/rnp/rnp.h

```diff
@@ -2226,8 +2232,8 @@ RNP_API rnp_result_t rnp_op_sign_destroy(rnp_op_sign_t op);
 /* Verification */
 
 /** @brief Create verification operation context. This method should be used for embedded
- *         signatures or cleartext signed data. For detached verification corresponding
- *         function should be used.
+ *         signatures, cleartext signed data or just encrypted data. For the detached signature
+ *         verification should be used function rnp_op_verify_detached_create().
```

Comment:
> For the detached signature  verification should be used function rnp_op_verify_detached_create().

Invalid syntax of the sentence. I think this way should be correct:

"For detached signature verification the function
rnp_op_verify_detached_create() should be used."

Reply:
Agree, thanks!